### PR TITLE
Ticket/13807 Allow building changesets for the event list

### PR DESCRIPTION
**Pull request by nickvergessen**
https://tracker.phpbb.com/browse/PHPBB3-13807

### CODE REVIEW

File: phpBB/phpbb/event/php_exporter.php [hunk 1]
```diff
@@ -215,6 +234,33 @@ public function crawl_php_file($file)
...
 
+					$changed_line_nums = $this->find_changed('changed');
+					if (empty($changed_line_nums))
+					{
+						$changed_line_nums = $this->find_changed('change');
+					}
+					$changed_versions = array();
+					if (!empty($changed_line_nums))
+					{
+						foreach ($changed_line_nums as $changed_line_num) {
```
Comment (Elsensee) [inline phpBB/phpbb/event/php_exporter.php]: brace should go on its own line ;)

Comment (nickvergessen) [inline phpBB/phpbb/event/php_exporter.php]: fixed

File: phpBB/develop/export_events_for_wiki.php [hunk 2]
```diff
@@ -65,10 +74,19 @@ function validate_argument_count($arguments, $count)
 	case 'all':
 		echo '__FORCETOC__' . "\n";
 
+	case 'diff':
+		if ($action === 'diff')
+		{
+			echo '== Event changes ==' . "\n";
+		}
+		$min_version = $extension;
```
Comment (nickvergessen) [inline phpBB/develop/export_events_for_wiki.php]: this needs moving aswell

**Comment by Nicofuma** [comment]
add [VERSION]to the usage please

**Comment by nickvergessen** [comment]
fixed

**Pull request merged**